bot.js

```diff
@@ -284,6 +284,43 @@ Discord.on('message', msg => {
     }
 });
 
+Discord.on('ready', () => {
+//This allows you to make a custom status that rotates!
+	let statuses = [
+    'Status 1'
+		,
+  'Status 2'
+		,
+      'Status 3'
+		,
+  'Status 4'
+		,
+'Status 5
+		',
+     'Status 6'
+		,
+	              'Status 7'
+		,
+	        'Status 8'
+		,
+	                                         'Status 9'
+		,
+	            'Status 10'
+		,
+       ];
+//You HAVE TO HAVE 10 STATUSES FOR THIS TO WORK	AND THE LAYOUT MUST REMAIN THE SAME
+	setInterval(
+		function() 
+		{
+           let status = 
+	       statuses[Math.floor(Math.random() * statuses.length)];
+           Discord
+		   .user
+		   	.setActivity(status, {type: "WATCHING"});
+       }, 8);

```

Comment:
this is not 8 seconds, it's 8 milliseconds :^)

Reply:
Untrue

Reply:
its 8 milliseconds

Reply:
8000 would be 8 seconds

Reply:
Ah shi let me add a package

Reply:
why would you need a package just to make it 8ms to 8000 :yikes:

Reply:
> this is not 8 seconds, it's 8 milliseconds :^)

I see no issue